boa3_test/test_drive/testrunner/blockchain/block.py

```diff
@@ -74,20 +74,31 @@ def to_json(self) -> Dict[str, Any]:
         return json_block
 
     @classmethod
-    def from_json(cls, json: Dict[str, Any]) -> TestRunnerBlock:
+    def from_json(cls, json: Dict[str, Any], *args, **kwargs) -> TestRunnerBlock:
```

Comment:
Why not define this function as `def from_json(cls, json: Dict[str, Any], *, neoxp_config:NeoExpressConfig=None) -> TestRunnerBlock:` instead?
This way neoxp_config would end up being a keyword argument only and it would be more explicit that you can send it as an argument

Reply:
since from_json is a overwritten method from base class, it has to have the same signature, that's why I had to include *args and **kwargs on both `TestRunnerBlock` and generic `TestBlock` to be able to pass neoxp_config as an additional arg

same for `TestRunnerTransaction` and `TestTransaction`